pkgs/top-level/all-packages.nix

```diff
@@ -700,6 +700,8 @@ in
 
   deskew = callPackage ../applications/graphics/deskew { };
 
+  detect-secrets = pythonPackages.callPackage ../development/tools/detect-secrets { };
```

Comment:
Why not `python3Packages`?

Reply:
no special reason, I wouldn't mind changing it if you prefer it

Reply:
I think it's preferred to use python3 when it's an application.